### PR TITLE
added bucket system and get NGS_Automated working on betabarrel

### PR DESCRIPTION
**Pull request by RoanKanninga**

### CODE REVIEW

File: bin/copyProjectDataToPrm.sh [hunk 1]
```diff
@@ -57,10 +57,12 @@ Usage:
...
 Options:
 
-	-h   Show this help.
-	-g   Group.
-	-n   Dry-run: Do not perform actual sync, but only list changes instead.
-	-l   Log level.
+	-h	Show this help.
+	-g	Group.
+	-p	[pipeline]
+		from which pipeline is the data coming from (NGS_Demultiplexing, GAP)
```
Comment (pneerincx) [inline bin/copyProjectDataToPrm.sh:63]: from ... from is a bit strange. How about:
`Pipeline that produced the project data that needs to be transferred to prm.`

File: bin/copyProjectDataToPrm.sh [hunk 2]
```diff
@@ -458,7 +466,7 @@ else
 				run=$(basename "${run}")
 				controlFileBase="${PRM_ROOT_DIR}/logs/${project}/${run}"
 				export JOB_CONTROLE_FILE_BASE="${controlFileBase}.${SCRIPT_NAME}"
-				calculateProjectMd5sFinishedFile="ssh ${DATA_MANAGER}@${HOSTNAME_TMP}:${TMP_ROOT_DIAGNOSTICS_DIR}/logs/${project}/${run}.calculateProjectMd5s.finished"
+				calculateProjectMd5sFinishedFile="ssh ${DATA_MANAGER}@${HOSTNAME_TMP}:${TMP_ROOT_DIAGNOSTICS_DIR}/logs//${project}/${run}.calculateProjectMd5s.finished"
```
Comment (pneerincx) [inline bin/copyProjectDataToPrm.sh:469]: Why the extra redundant `/`

Comment (RoanKanninga) [reply bin/copyProjectDataToPrm.sh:469]: i dunno

File: bin/copyProjectDataToPrm.sh (hunk 1 above)
Comment (RoanKanninga) [reply bin/copyProjectDataToPrm.sh:63]: fixed

**Review by pneerincx — APPROVED** [review]

**Pull request merged**